.circleci/docker-compose-prod.yml

```diff
@@ -1,39 +1,68 @@
-version: '3'
+version: '3.1'
+
 services:
   couch:
     image: "couchbase:5.1.0"
-    ports:
-      - "8091-8094:8091-8094"
-      - "11210-11211:11210-11211"
-    container_name: "couch-dakar"
     volumes:
       - couch:/opt/couchbase/var
     networks:
-      - dakar-network
-  app:
+      #- dakar-network
+      - traefik_traefik
+    deploy:
+      labels:
+        - "traefik.enable=true"
+        - "traefik.http.routers.dakar-couch.rule=Host(`dakar-couch.project.lambla.eu`)"
+        - "traefik.http.routers.dakar-couch.entrypoints=web"
+        - "traefik.http.services.dakar-couch.loadbalancer.server.port=8091"
+
+  api:
     image: "dakarinternational/dakar:latest"
-    ports:
-      - "8080:8080"
-    container_name: 'app-dakar'
     depends_on:
       - "couch"
     networks:
-      - dakar-network
+      #- dakar-network
+      - traefik_traefik
     environment:
```

Comment:
how does it work?

Reply:
so, for now, all our services are on the same network. 
just the time for me to figure out how to manage multiple networks properly, without having Gateway Timeout exceptions...